### ISSUE TITLE
initialSlide not working

### ISSUE THREAD

**Issue by PitySOFT** [vasyabigi/angular-slick#128]
To set up the initial slide this has to be type int, at this moment this is set up as a string so this will brake the angular slick.

Please replace line 97 with this

``` JavaScript
initialSlide: parseInt(scope.initialSlide) || 0,
```

**Comment by jurrinus** [comment]
Yes, I just came across a similar issue and the proposed fix PitySOFT fixed the problem.

**Comment by ashish173** [comment]
Any chances of this going into release anytime soon ? @vasyabigi ?
btw @jurrinus @PitySOFT how can I change this line when its in the library and angular-slick is downloaded by grunt will it not override in production mode?

**Comment by lewissmallmedia** [comment]
Yes please I have the same problem as @ashish173. When the server pulls everything together it will break.

**Comment by nikhil-batra** [comment]
Still having this problem.
